app/plugin_api.go

```diff
@@ -1063,3 +1063,22 @@ func (api *PluginAPI) DeleteCommand(commandID string) error {
 
 	return nil
 }
+
+// PublishPluginEvent broadcasts a plugin event to all other running instances of
+// the calling plugin.
+func (api *PluginAPI) PublishPluginEvent(ev model.PluginEvent, opts model.PluginEventSendOptions) error {
+	if api.app.Cluster() == nil {
+		return nil
+	}
```

Comment:
We may decide to return an error in this case. Keeping it a no-op for the moment.